data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
+            link: "http://hamburg-muss-handeln.de/",
+            keywords: ["private-sector", "legislation", "local", "grassroots"],
+            label: "hamburg-muss-handeln",
+        },
+        description: {
+            content: "A coalition of 300+ companies and organizations from Hamburg to express support for the demands of Fridays for Future. Also provides advice for upcoming state elections.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Fossil Free",
+            link: "https://gofossilfree.org/",
+            keywords: ["grassroots", "legislation", "private-sector", "network", "ngo"],
+            label: "fossil-free",
+        },
+        description: {
+            content: "A network of local grassroots initiatives that individually petition governments and companies to go fossil free. Local groups are present in virtually all major city and are independent in how they pursue their goals and who they talk to.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2012](https://gofossilfree.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Scientists for Future",
+            link: "https://www.scientists4future.org/",
+            keywords: ["science", "grassroots", "education", "ngo"],
+            label: "scientists-for-future",
+        },
+        description: {
+            content: "An interdisciplinary group of scientists working for a sustainable future. They support Fridays for Future and established [Lectures for Future](https://www.scientists4future.org/infomaterial/vortragsreihen/) to push the climate crisis onto the agenda in universities.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "Probably [2019](https://www.scientists4future.org/about/faq/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Europe Beyond Coal",
```

Comment:
There is also Beyond Coal Australia https://www.beyondcoal.org.au/
Same Problem with '... handelt'.
I guess a whole new organization in the script would be useful. Please also check for other regional initiatives.

Reply:
I added it to [Content to Add > Initiatives](https://docs.google.com/spreadsheets/d/1mN4i9pIJuqvhFS1s-EB4l4ICs7gmW8gZzQyqH8jJD8Y/edit#gid=267342746), let's collect new things to add there

Reply:
Great!